pkg/cableengine/ipsec/strongswan.go

```diff
@@ -291,15 +312,52 @@ func (i *strongSwan) loadSharedKey(endpoint types.SubmarinerEndpoint, client *go
 	return nil
 }
 
-func runCharon(debug bool, logFile string) error {
+const charonConfTemplate = `
+	charon {
+		port = {{.ipSecIKEPort}}
+		port_nat_t = {{.ipSecNATTPort}}
+		make_before_break = yes
```

Comment:
(nv) do you want to perhaps comment on why we enable these two options?

Reply:
I have absolutely no idea... may be it's worth investigating them, they were here before I touched this :), although, they were ineffective as we saw because they were installed on the wrong place.

Reply:
yeah, I know. But now that we are "introducing" them in a new form it might be worth checking why it's there in the first place

Reply:
It seems like make-before-break is related to maintaining dataplane connectivity while re-authenticating an endpoint with IKEv2.

For the other one... I dont comprehend... https://github.com/strongswan/strongswan/blob/master/conf/options/charon.opt#L152

@Oats87  ^ any idea on what ignore_acquire_ts means?